### PR TITLE
Removing function not used for nifgen

### PR DESCRIPTION
**Pull request by injaleea**
[X] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md).

~~[ ] I've updated [CHANGELOG.md](https://github.com/ni/nimi-python/blob/master/CHANGELOG.md) if applicable.~~

~~[ ] I've added tests applicable for this pull request~~

### What does this Pull Request accomplish?
Removing GetAttributeViInt64, SetAttributeViInt64 from nifgen since there is no ViInt64 attributes.

Submission of this PR will be based on the decision for issue #608 

### What testing has been done?

systemtests, tox

### CODE REVIEW

**Comment by injaleea** [comment]
test this please

File: src/nifgen/metadata/functions_addon.py [hunk 1]
```diff
@@ -57,6 +57,7 @@
     '.+Complex.+':                          { 'codegen_method': 'no',       },
     'GetStreamEndpointHandle':              { 'codegen_method': 'no',       },
     'AdjustSampleClockRelativeDelay':       { 'codegen_method': 'no',       },  # This is used internally by NI-TClk, but not by end users.
+	'.etAttributeViInt64':                  { 'codegen_method': 'no',       },  # Not used by python APIs
```
Comment (texasaggie97) [inline src/nifgen/metadata/functions_addon.py]: Please fix the formatting

File: src/nifgen/metadata/functions_addon.py [hunk 2]
```diff
@@ -57,7 +57,7 @@
     '.+Complex.+':                          { 'codegen_method': 'no',       },
     'GetStreamEndpointHandle':              { 'codegen_method': 'no',       },
     'AdjustSampleClockRelativeDelay':       { 'codegen_method': 'no',       },  # This is used internally by NI-TClk, but not by end users.
-	'.etAttributeViInt64':                  { 'codegen_method': 'no',       },  # Not used by python APIs
+    '.etAttributeViInt64':                  { 'codegen_method': 'no',       },  # Not used by python APIs
```
Comment (marcoskirsch) [inline src/nifgen/metadata/functions_addon.py]: Comment is inaccurate.
Should be something like "NI-FGEN has no ViInt64 attributes".

Comment (marcoskirsch) [reply src/nifgen/metadata/functions_addon.py]: Fixed in master.

**Pull request merged**